### DISCUSSION TITLE
Error in line 1

### DISCUSSION THREAD

**Discussion by BiggDaddyy** [LSPosed/MagiskOnWSALocal#425] [Q&A] [answered]
i'm getting this Error and i tried to build it again like 6 times xD 

C:\WSA\Install.ps1 : Some files are missing in the folder. Please try to build again. Press any key to exist
At line:1 char:24
+ Set-Location 'C:\WSA'; &'C:\WSA\Install.ps1'
+                        ~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Install.ps1

**Comment by BiggDaddyy** [comment]
C:\WSA_2301.40000.7.0_x64_Release-Nightly-MindTheGapps-13.0\Install.ps1 : Some files are missing in the folder. Please
try to build again. Press any key to exist
At line:1 char:77
+ ... apps-13.0'; &'C:\WSA_2301.40000.7.0_x64_Release-Nightly-MindTheGapps- ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Install.ps1

**Reply by MRARPG** [reply]
Estou com esse erro e não  consigo instalar ou resolver

**Comment by Howard20181** [comment, marked as answer]
A fix was just pushed, please update

**Reply by BiggDaddyy** [reply]
ok Thanks <3

**Comment by BiggDaddyy** [comment]
Installing MagiskOnWSA...
Add-AppxPackage : Deployment failed with HRESULT: 0x80073CF6, The package was not registered.
                                                                                                                         Deployment operation progress: C:\WSA_2301.40000.7.0_x64_Release-Nightly-with-magisk-25.2(25200)-stable-MindTheGapps-13    Processing                                                                                                              [oooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo                                      ]                                                                                                                              + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : WriteError: (C:\WSA_2301.400...ppxManifest.xml:String) [Add-AppxPackage], IOException
    + FullyQualifiedErrorId : DeploymentError,Microsoft.Windows.Appx.PackageManager.Commands.AddAppxPackageCommand

All Done!
Press any key to exit

is what i'm getting now xD
